product_docs/docs/biganimal/release/overview/02_high_availability/distributed_highavailability.mdx

```diff
@@ -0,0 +1,70 @@
+
+The witness node/witness group doesn't host data but exists for management purposes. It supports operations that require a consensus, for example, in case of an availability zone failure. 
+
+!!!Note
+   Operations against a distributed high-availability cluster leverage the [EDB Postgres Distributed switchover](/pgd/latest/cli/command_ref/pgd_switchover/) feature, which provides subsecond interruptions during planned lifecycle operations.
+
+## Single data location 
+
+A configuration with single data location has one data group and either:
+
+- Two data nodes with one lead and one shadow and a witness node each in separate availability zones
+
+    ![region(2 data + 1 witness)](../images/image5.png)
+
+- Three data nodes with one lead and two shadow nodes each in separate availability zones
+
+    ![region(3 data)](../images/image3.png)
+
+## Multiple data locations and witness node
+
+A configuration with multiple data locations has two data groups that contain either:
+
+- Two data groups (not recommended for production):
+
+  - A data node, shadow node, and a witness node in one region
+  - The same configuration in another region
+  - A witness node in a third region
+
+    ![region(2 data + 1 shadow) + region(2 data + 1 shadow) + region(1 witness)](../images/2dn-1wn-2dn-1wn-1wg.png)
+
+- Three data groups:
+
+  - A data node and two shadow nodes in one region
+  - The same configuration in another region
+  - A witness node in a third region
+
+    ![region(2 data + 1 shadow) + region(2 data + 1 shadow) + region(1 witness)](../images/eha.png)
+
+
+### Cross-cloud service providers (CSP) witness node
+
+By default, the cloud service provider selected for the data groups is preselected for the witness node.
+
+To further improve availability, you can designate a witness node on a different cloud service provider than the data groups. This configuration can enable a three-region configuration even if a single cloud provider only offers two regions in the jurisdiction you are allowed to deploy your cluster in.
+
+You can also change the cloud service provider of the witness node to be one of the other two cloud service providers.
+
+Cross-cloud service provider witness nodes are available with AWS, Azure, and Google Cloud using your own cloud account and BigAnimal's cloud account. This option is enabled by default and applies to both multi-region configurations available with PGD. The database price isn't charged for witness nodes but for the infrastructure it uses. The cost of the cross-CSP witness node is reflected in the pricing estimate at the bottom of your BigAnimal portal.
```

Comment:
@nidhibhammar let's change "The database price isn't charged for witness nodes but for the infrastructure it uses. The cost of the cross-CSP witness node is reflected in the pricing estimate at the bottom of your BigAnimal portal." to something like "For witness nodes you only pay for the infrastructure used, which is reflected in the pricing estimate."

we shouldn't specify where things are in the portal too much since they could change, which would then make the docs inaccurate